exporter/lokiexporter/exporter.go

```diff
@@ -179,11 +194,24 @@ func (l *lokiExporter) convertAttributesToLabels(attributes pdata.AttributeMap)
 		}
 	}
 
-	if len(ls) == 0 {
-		return nil, false
+	return ls
+}
+
+func (l *lokiExporter) convertResourceAttributesToLabels(attributes pdata.AttributeMap) model.LabelSet {
```

Comment:
Any change you had thoughts on combining these methods too? Seems to be a bit too much duplication if we can avoid it.

Reply:
I suppose I could add a sub struct for all labels (`map[string]ToLabels`) onto `lokiExporter` struct and have both logResourceAttribute labels and ResourceAttribute labels on that. This would mean I could merge the functions and would also work with the above duplicate code?